policy/release/buildah_build_task.rego

```diff
@@ -9,33 +9,7 @@ package policy.release.buildah_build_task
 import rego.v1
 
 import data.lib
-
-# METADATA
-# title: Buildah task has Dockerfile param defined
```

Comment:
This one was first introduced to safe guard against a builder Task not including the `DOCKERFILE` parameter. For example, if the parameter is renamed, `buildah_uses_local_dockerfile` is no-op. Unlikely, but it could happen.

Maybe we want to introduce a new policy rule under the `task` directory to verify this, and enable it [here](https://github.com/konflux-ci/build-definitions/tree/main/policies). 

WDYT?

Reply:
Not sure, not all image-building tasks need a Dockerfile, e.g. the s2i ones. I think we'd need to classify which Tasks need a `DOCKERFILE` parameter and which do not, and I choose the simplest possible classifications: those that do have the `DOCKERFILE` parameter. Making it very trivial and pragmatic.
I'm don't know how strict is the hermetic build requirement, I'm expecting folk to run only hermetic releases in the future, so a Dockerfile from remote should not be possible, and this rule might not be needed at all...

Reply:
WFM!